### PR TITLE
PLAT-1373 | Unify spacings and some styles between Quill and displayed HTML

### PR DESCRIPTION
**Pull request by kvas-damian**

### CODE REVIEW

File: resources/assets/js/admin/components/quizes/edit/QuizQuestionsSetEditor.vue [hunk 1]
```diff
@@ -37,6 +37,7 @@
 			</span>
 			<label class="label">Opis</label>
 			<wnl-quill
+				class="content"
 				ref="descriptionEditor"
```
Comment (rogatty) [inline resources/assets/js/admin/components/quizes/edit/QuizQuestionsSetEditor.vue]:
```
/home/travis/build/bethinkpl/wnl-platform/resources/assets/js/admin/components/quizes/edit/QuizQuestionsSetEditor.vue
  41:5  error  Attribute "ref" should go before "class"  vue/attributes-order
```

File: database/migrations/2019_06_14_121958_cleanup_empty_lines_in_quiz_questions.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	 */
+	public function up()
+	{
+		\DB::statement("update quiz_questions set text = replace(text, '<p><br></p>', '');");
+		\DB::statement("update quiz_questions set text = replace(text, '<p class=\"ql-indent-1\"><br></p>', '');");
+		\DB::statement("update quiz_questions set text = replace(text, '<p class=\"ql-indent-2\"><br></p>', '');");
+		\DB::statement("update quiz_questions set explanation = replace(explanation, '<p><br></p>', '');");
+
+
+		\DB::statement("update comments set text = replace(text, '<p><br></p>', '');");
```
Comment (rogatty) [inline database/migrations/2019_06_14_121958_cleanup_empty_lines_in_quiz_questions.php]: @kvas-damian what about `qna_questions` and `qna_answers`?

File: database/migrations/2019_06_14_121958_cleanup_empty_lines_in_quiz_questions.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CleanupEmptyLinesInQuizQuestions extends Migration
```
Comment (rogatty) [inline database/migrations/2019_06_14_121958_cleanup_empty_lines_in_quiz_questions.php]: The name is outdated.

File: database/migrations/2019_06_14_121958_cleanup_empty_lines_in_quiz_questions.php (hunk 2 above)
Comment (rogatty) [reply database/migrations/2019_06_14_121958_cleanup_empty_lines_in_quiz_questions.php]: Ah, I see we override Bulma margins in: https://github.com/bethinkpl/wnl-platform/blob/4f7d602cf69c55aad16f53d71f87c7e6f259e7e3/resources/assets/styles/_styleguide.scss#L367-L378
Why don't we unify and clean up those too?

Comment (rogatty) [reply database/migrations/2019_06_14_121958_cleanup_empty_lines_in_quiz_questions.php]: Actually, `qna-question-content` class isn't used anymore so there are now large gaps.

Comment (rogatty) [reply database/migrations/2019_06_14_121958_cleanup_empty_lines_in_quiz_questions.php]: On `db-lek-1`:
```
mysql> select count(1) from qna_questions where text like '%<p><br></p>%';
+----------+
| count(1) |
+----------+
|       92 |
+----------+
1 row in set (0.00 sec)

mysql> select count(1) from qna_answers where text like '%<p><br></p>%';
+----------+
| count(1) |
+----------+
|      192 |
+----------+
1 row in set (0.00 sec)
```

File: lib/Bethink/Bethink.php [hunk 4]
```diff
@@ -80,4 +80,18 @@ public function mbSubstrReplace($string, $replacement, $start, $length = null, $
 			return mb_substr_replace($string, $replacement, $start, $length, $encoding);
 		}
 	}
+
+	// https://www.php.net/manual/en/function.rtrim.php#95802
+	public function strrtrim(string $message, string $strip): string
```
Comment (kvas-damian) [inline lib/Bethink/Bethink.php]: can we have some tests for it?

File: app/Models/QnaQuestion.php [hunk 5]
```diff
@@ -57,6 +58,11 @@ public function reactions()
 		return $this->morphToMany('App\Models\Reaction', 'reactable');
 	}
 
+	public function getTextAttribute()
+	{
+		return Bethink::strrtrim($this->attributes['text'], '<p><br></p>');
```
Comment (kvas-damian) [inline app/Models/QnaQuestion.php]: Instead of fixing it while we query data from the DB, maybe it's better to do that on save?

**Pull request merged**